### PR TITLE
LPS-200111 dm copy to error message

### PR DESCRIPTION
**Pull request by ambrinchaudhary**
Issue: https://liferay.atlassian.net/browse/LPS-200111 
Now the check in the destination site have a different error message. 

Screenshot 2023-10-27 at 12 05 14

### CODE REVIEW

**Comment by ambrinchaudhary** [comment]
ci:test:sf

**Comment by boton** [comment]
ci:forward

**Comment by SylviaLuan** [comment]
Hi @ambrinchaudhary,

Please fix rebase errors and then I will continue to analyze this PR and forward. Thanks.

**Comment by ambrinchaudhary** [comment]
ci:test:sf

**Comment by ambrinchaudhary** [comment]
ci:forward

File: modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/DLCopyValidationUtil.java [hunk 1]
```diff
@@ -16,13 +16,22 @@ public class DLCopyValidationUtil {
 
 	public static String getCopyToValidationMessage(
 		long companyMaxSizeToCopy, long groupMaxSizeToCopy,
-		long systemMaxSizeToCopy, long size, Locale locale) {
+		long systemMaxSizeToCopy, long size, Locale locale,
+		boolean checkInDestination) {
```
Comment (adolfopa) [inline modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/DLCopyValidationUtil.java:20]: This reads confusing, as it has nothing to do with the "check in" operation. Maybe something like `targetGroupValidationFailed` could work?

**Review by adolfopa — CHANGES_REQUESTED** [review]

File: modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/DLCopyValidationUtil.java (hunk 1 above)
Comment (ambrinchaudhary) [reply modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/DLCopyValidationUtil.java]: I renamed it to `targetGroupValidation`

**Comment by ambrinchaudhary** [comment]
ci:forward

**Comment by SylviaLuan** [comment]
ci:forward:force

**Pull request closed without merging**